Please fix documentation errors

When using Help -> Help Contents and then searching for a arbitrary string, all the pages are loaded and indexed which leads to a bunch of errors, many of which are in the rap documentation:

<img width="1068" height="1324" alt="Image" src="https://github.com/user-attachments/assets/2d51ff02-d735-4056-b6de-9fb06d87c52a" />

I expect you can reproduce this with a self-host launch from the RAP Tools development IDE.

I tested the latest version and it's better but it seems one problem still shows up.  I didn't investigate in detail.  Too much to do.

`Exception occurred while adding document /org.eclipse.rap.doc/guide/reference/jsdoc/symbols/rap.html to index.`

```
java.io.IOException: D:\Users\test40\.p2\pool\plugins\org.eclipse.rap.doc_4.7.0.20260529-1200.jar
	at org.eclipse.osgi.storage.bundlefile.CloseableBundleFile$BundleEntryInputStream.enrichExceptionWithBaseFile(CloseableBundleFile.java:591)
	at org.eclipse.osgi.storage.bundlefile.CloseableBundleFile$BundleEntryInputStream.read(CloseableBundleFile.java:546)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLEntityManager$RewindableInputStream.readAndBuffer(XMLEntityManager.java:3092)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLEntityManager.setupCurrentEntity(XMLEntityManager.java:718)
	at java.xml/com.sun.org.apache.xerces.internal.impl.XMLVersionDetector.determineDocVersion(XMLVersionDetector.java:150)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:862)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:826)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:137)
	at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1225)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:629)
	at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(SAXParserImpl.java:328)
	at java.xml/javax.xml.parsers.SAXParser.parse(SAXParser.java:197)
	at org.eclipse.help.search.SearchParticipantXML.addDocument(SearchParticipantXML.java:301)
	at org.eclipse.help.internal.search.HTMLSearchParticipant.addDocument(HTMLSearchParticipant.java:57)
	at org.eclipse.help.internal.search.SearchIndex.addDocument(SearchIndex.java:294)
	at org.eclipse.help.internal.search.IndexingOperation.addDocuments(IndexingOperation.java:249)
	at org.eclipse.help.internal.search.IndexingOperation.addNewDocuments(IndexingOperation.java:160)
	at org.eclipse.help.internal.search.IndexingOperation.execute(IndexingOperation.java:109)
	at org.eclipse.help.internal.search.LocalSearchManager.updateIndex(LocalSearchManager.java:652)
	at org.eclipse.help.internal.search.LocalSearchManager.ensureIndexUpdated(LocalSearchManager.java:626)
	at org.eclipse.help.internal.search.LocalSearchManager.search(LocalSearchManager.java:571)
	at org.eclipse.help.internal.search.SearchManager.searchLocal(SearchManager.java:108)
	at org.eclipse.help.internal.search.SearchManager.search(SearchManager.java:99)
	at org.eclipse.help.internal.search.SearchProgressMonitor.lambda$0(SearchProgressMonitor.java:180)
	at java.base/java.lang.Thread.run(Thread.java:1474)
Caused by: java.io.IOException: Stream closed
	at java.base/java.util.zip.InflaterInputStream.ensureOpen(InflaterInputStream.java:85)
	at java.base/java.util.zip.InflaterInputStream.read(InflaterInputStream.java:155)
	at java.base/java.io.FilterInputStream.read(FilterInputStream.java:71)
	at org.eclipse.osgi.storage.bundlefile.CloseableBundleFile$BundleEntryInputStream.read(CloseableBundleFile.java:544)
	... 23 more
```

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/$.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/$.html
@@ -493,7 +493,6 @@ a plain object with key-value pairs to set.</dd>
               <dt class="heading">Since:</dt>
                 <dd>3.0</dd>
               </dl>
-              </dl>
             
             
             
@@ -531,7 +530,6 @@ string only. Otherwise a reference to this object.</dd>
               <dt class="heading">Since:</dt>
                 <dd>3.0</dd>
               </dl>
-              </dl>
             
             
             
@@ -558,7 +556,7 @@ string only. Otherwise a reference to this object.</dd>
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Button.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Button.html
@@ -673,7 +673,7 @@ it is selected when it is pushed in.
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Canvas.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Canvas.html
@@ -463,7 +463,7 @@ ul.inheritsList
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Client.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Client.html
@@ -1261,7 +1261,7 @@ drop-shadow and linear-gradient.
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Combo.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Combo.html
@@ -749,7 +749,7 @@ to display incorrectly.
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Composite.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Composite.html
@@ -563,7 +563,7 @@ ul.inheritsList
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Control.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Control.html
@@ -1862,7 +1862,7 @@ detached from the widget hierarchy and removed from DOM
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Event.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Event.html
@@ -858,7 +858,7 @@ Set for MouseDown, MouseUp, MouseMove, MouseEvnet, MouseExit and MouseDoubleClic
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Label.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Label.html
@@ -502,7 +502,7 @@ ul.inheritsList
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/ProgressBar.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/ProgressBar.html
@@ -584,7 +584,7 @@ must be greater than or equal to zero.
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/RemoteObject.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/RemoteObject.html
@@ -550,7 +550,7 @@ written to the message.
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/SWT.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/SWT.html
@@ -3604,7 +3604,7 @@ Notes:
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Scale.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Scale.html
@@ -655,7 +655,7 @@ must be greater than or equal to zero.
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/ScrolledComposite.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/ScrolledComposite.html
@@ -515,7 +515,7 @@ ul.inheritsList
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Slider.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Slider.html
@@ -655,7 +655,7 @@ must be greater than or equal to zero.
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Spinner.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Spinner.html
@@ -752,7 +752,7 @@ must be greater than or equal to zero.
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Text.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Text.html
@@ -834,7 +834,7 @@ operation is undefined and may vary from platform to platform.
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Widget.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/Widget.html
@@ -556,7 +556,7 @@ with the specified name to the given value.
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/rap.html
+++ b/bundles/org.eclipse.rap.doc/guide/reference/jsdoc/symbols/rap.html
@@ -500,7 +500,6 @@ by <code>org.eclipse.rap.rwt.widget.WidgetUtil.getId</code> and
               <dt class="heading">Since:</dt>
                 <dd>3.2</dd>
               </dl>
-              </dl>
             
             
             
@@ -877,7 +876,7 @@ sent for types that are given in this list and are listened to by the server. Se
 
 <!-- ============================== footer ================================= -->
     <div class="fineprint" style="clear:both">
-      Copyright (c) EclipseSource and others 2013, 2022. All rights reserved.
+      Copyright (c) EclipseSource and others 2013, 2026. All rights reserved.
     </div>
   </body>
 </html>

--- a/bundles/org.eclipse.rap.doc/guide/toc.xml
+++ b/bundles/org.eclipse.rap.doc/guide/toc.xml
@@ -160,8 +160,6 @@
           href="guide/reference/extension-points/org_eclipse_rap_ui_entrypoint.html"/>
       <topic label="org.eclipse.rap.ui.layouts"
           href="guide/reference/extension-points/org_eclipse_rap_ui_layouts.html"/>
-      <topic label="org.eclipse.rap.ui.phaselistener"
-          href="guide/reference/extension-points/org_eclipse_rap_ui_phaselistener.html"/>
       <topic label="org.eclipse.rap.ui.resources"
           href="guide/reference/extension-points/org_eclipse_rap_ui_resources.html"/>
       <topic label="org.eclipse.rap.ui.serviceHandler"

